src/libcore/marker.rs

```diff
@@ -88,7 +91,12 @@ impl<T: ?Sized> !Send for *mut T { }
 /// [trait object]: ../../book/first-edition/trait-objects.html
 #[stable(feature = "rust1", since = "1.0.0")]
 #[lang = "sized"]
-#[rustc_on_unimplemented = "`{Self}` does not have a constant size known at compile-time"]
+#[rustc_on_unimplemented(
+    message="the size for value values of type `{Self}` cannot be known at compilation time",
```

Comment:
is "the size for value values" a typo here ?

Reply:
It is. Can you file a ticket (or even better, a quick PR)?

Reply:
I've opened https://github.com/rust-lang/rust/pull/52231 to fix it